lua/nvim-lsp-installer/servers/clangd.lua

```diff
@@ -0,0 +1,30 @@
+local server = require('nvim-lsp-installer.server')
+
+local root_dir = server.get_server_root_path('c-family')
+
+local install_cmd = [=[
+rm -r clangd;
+
+if [[ $(uname) == Linux ]]; then
+  wget -O clangd.zip https://github.com/clangd/clangd/releases/download/11.0.0/clangd-linux-11.0.0.zip;
+elif [[ $(uname) == Darwin ]]; then 
+  wget -O clangd.zip https://github.com/clangd/clangd/releases/download/11.0.0/clangd-mac-11.0.0.zip; 
+else 
+  >&2 echo "$(uname) not supported."; 
+  exit 1;
+fi
+
+unzip clangd.zip; 
+rm clangd.zip;
+mv clangd_11.0.0 clangd;
+
+]=]
+
+return server.Server:new {
+  name = "clangd",
+  root_dir = root_dir, 
+  install_cmd = install_cmd, 
+  default_options = {
+    cmd = { root_dir .. '/clangd/bin/clangd'},
```

Comment:
It seems like [LSP config](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/clangd.lua#L30) provides a `--background-index` argument as well. I can't seem to find any docs or traces of it in clangd itself though, do you happen to know what it could be?

Reply:
from `clangd --help`: 
`  --background-index              - Index project code in the background and persist index on disk.`

Reply:
Why this argument has not been added to default configuration, sounds useful

Reply:
Oh, had someone asked me I'd have thought it was added already. Happy to receive PR that adds it!